src/components/Login/Login.js

```diff
@@ -1,25 +1,80 @@
-import React from "react";
-import { Modal, Col, Row, Container } from "react-bootstrap";
+import React, { useState } from "react";
+import { Modal, Col, Row, Container, Alert } from "react-bootstrap";
 import "./Login.css";
 import { FaFacebook, FaPaw } from "react-icons/fa";
 import { ImGoogle3 } from "react-icons/im";
 import { MdClose } from "react-icons/md";
 import { Formik, Field, Form, ErrorMessage } from "formik";
 import * as Yup from "yup";
+import { useDispatch } from "react-redux";
+import { SIGN_IN } from "../../store/actions/actionConst";
+import {
+  signInWithGoogle,
+  signInWithFacebook,
+  signInWithEmailAndPassword,
+} from "../../helpers/auth";
+
+function Login({ show, showModal }) {
+  const [errorMsg, setErrorMsg] = useState(false);
+  const dispatch = useDispatch();
+
+  const signWithSocial = async (social) => {
+    let fnCall = null;
+    if (social === "google") {
+      fnCall = signInWithGoogle;
+    } else if (social === "facebook") {
+      fnCall = signInWithFacebook;
+    }
+    if (fnCall) {
```

Comment:
looking are previous comment that means that we dont need this if condidtion

Reply:
Done